transport/UdpEndpointImpl.cpp

```diff
@@ -29,7 +29,7 @@ UdpEndpointImpl::UdpEndpointImpl(jobmanager::JobManager& jobManager,
     bool isShared)
     : BaseUdpEndpoint("UdpEndpoint", jobManager, maxSessionCount, allocator, localPort, epoll, isShared),
       _iceListeners(maxSessionCount * 2),
-      _dtlsListeners(maxSessionCount * 2),
+      _dtlsListeners(maxSessionCount * 5),
```

Comment:
Maybe discuss this. I reasoned that SMB sees only successful candidates here. Client may have many but usually only a couple of them will actually reach SMB. So even if a few clients do have more, most will connect with one successful candidate. Makes sense?

Reply:
Yes it make sense, but if will have successful candidates only,  isn't the `_dtlsListeners` most likely smaller then `_iceListeners`? Perhaps  I misunderstood it

Reply:
The iceListener is listening for user id, pwd which is one per user.
The dtls Listener can be multiple per user as it is one per candidate.